setup.cfg

```diff
@@ -0,0 +1,9 @@
+[flake8]
+max-line-length = 250
```

Comment:
oh lol I see we moved the lines stuff here

Reply:
Yup, this way it also runs with the right config when you do `flake8` without arguments.